src/components/datepicker/js/datepickerDirective.js

```diff
@@ -338,6 +330,25 @@
     /** @type {Number} Extra margin for the top of the floating calendar. Gets determined on the first open. */
     this.topMargin = null;
 
+    /** @type {!MdPanelPosition} Position object for the mdPanel instance. */
+    this.panelPosition = $mdPanel.newPanelPosition()
+      .relativeTo(this.inputContainer)
+      .addPanelPosition($mdPanel.xPosition.ALIGN_START, $mdPanel.yPosition.ALIGN_TOPS)
+      .withOffsetX(-this.leftMargin + 'px')
+      .withOffsetY(angular.bind(this, this.getPanelYOffset));
+
```

Comment:
Cool, I see it know... (window.addEventListener(this.windowEventName, this.windowEventHandler) ). Thank you.